src/Compatibility/Core/src/AppHostBuilderExtensions.cs

```diff
@@ -73,15 +73,6 @@ public static MauiAppBuilder UseMauiCompatibility(this MauiAppBuilder builder)
 				.ConfigureMauiHandlers(handlers =>
 				{
 #if PLATFORM
-
-#if !WINDOWS
-#if !(MACCATALYST || MACOS)
-#endif
-#else
-#pragma warning disable CS0618 // Type or member is obsolete
-					handlers.TryAddCompatibilityRenderer(typeof(Layout), typeof(LayoutRenderer));
```

Comment:
I'm thinking the initial bug here occurred when we did fully qualify this `Layout` type with the compatibility layout. 

If we change this to `Microsoft.Maui.Controls.Compatibility.Layout` and register that against `LayoutRenderer` that will still fix the issue yea?

Reply:
The Layout is already referenced from `Microsoft.Maui.Controls.Compatibility`. Fully qualifying the Layout type with its namespace does not resolve the issue.

Reply:
It should still resolve the ContentView to the `ContentViewHandler` though

That should still win out in this scenario. 

The non compatibility handlers should be picked first